config/200-mt-pingsource-clusterrole.yaml

```diff
@@ -0,0 +1 @@
+core/roles/mt-pingsource-clusterrole.yaml
```

Comment:
<!--whitespace-bot-->
Add trailing newline:
```suggestion
core/roles/mt-pingsource-clusterrole.yaml

```

Reply:
@mattmoor the bot shouldn't check symlinks, right?